gettsim/pensions.py

```diff
@@ -49,22 +49,24 @@ def update_earnings_points(person, tb, tb_pens):
 
     """
 
-    out = _ep_for_earnings(person, tb, tb_pens)
-    out += _ep_for_care_periods(person, tb, tb_pens)
+    out = _ep_for_earnings(person, tb, tb_pens, yr)
+    out += _ep_for_care_periods(person, tb, tb_pens, yr)
     # Note: We might need some interaction between the two
     # ways to accumulate earnings points (e.g., how to
     # determine what constitutes a 'care period')
     person["EP"] += out
     return person
 
 
-def _ep_for_earnings(person, tb, tb_pens):
+def _ep_for_earnings(person, tb, tb_pens, yr):
     """Return earning points for the wages earned in the last year."""
     westost = "o" if person["east"] else "w"
-    return np.minimum(person["m_wage"], tb["rvmaxek" + westost]) / tb_pens["meanwages"]
+    return (
```

Comment:
Do you need `np.minimum`? Are the entries vectors and you need the minimum element-wise?

Do not calculate on return.